wfl/Models/FoodOrderStatusDto.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Symphono.Wfl.Models
+{
+    public class FoodOrderStatusDto
+    {
+        public string Status { get; set; }
```

Comment:
If that should be applied to all the DTO's - how should I handle situations like the RestaurantDto when creating a restaurant - where all the checks necessary pertain to what is in that DTO (i.e. there is no entity in which I could move the logic to)?